src/components/generic/CallToActionBanner.vue

```diff
@@ -1,9 +1,14 @@
 <template>
-  <div class="home-cta d-lg-flex flex-lg-row flex-column">
-    <div class="cta-illustration align-self-stretch flex-lg-fill">
-      <div class="geometry" />
-    </div>
-    <div class="cta-content align-self-stretch flex-lg-fill">
+  <div
+    class="home-cta d-flex flex-md-row flex-column"
+    :class="variant"
+  >
+    <div
+      v-if="illustration"
+      class="cta-illustration align-self-stretch"
+      :style="`background-image: url(${illustration.image.url})`"
```

Comment:
This needs to go through our image optimisation helper function, in case images other than SVGs are uploaded and need to be scaled down to a reasonable maximum size.